keyboards/abko/ak84bt/info.json

```diff
@@ -123,8 +123,8 @@
     },
     "usb": {
         "device_version": "0.0.1",
-        "pid": "0x4321",
-        "vid": "0x7654"
+        "pid": "0x0011",
+        "vid": "0x369B"
```

Comment:
This vendor ID is already in use: 
https://www.the-sz.com/products/usbid/index.php?v=0x369B&p=&n=

Reply:
Sure. We are Iton Technology Corp and have obtained authorization from the USB Interface Alliance's to use this VID 0x369B, so future products will use this VID and a different PID for each model of keyboard. Many thanx!